dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/patch/ItemPatch.java

```diff
@@ -18,6 +18,8 @@
 
 /**
  * Provides PATCH operations for item updates.
+ *
+ * @author Michael Spalti
```

Comment:
I don't think this is an intentional change

Reply:
Hi @abollini ,
This was intentional (I had left off the author).  BTW, I don't need the attribution, but I assume the author tag is recommended as a way of providing work history.  Let me know if that's not the case and I'll make less use of it. :-)

Reply:
and...now that I've rebased I see that @cwilper has been working here as well. I'm still new to the process so please advise on what is most useful to everyone vis-a-vis author annotations!

Reply:
https://wiki.duraspace.org/display/DSPACE/Code+Style+Guide#CodeStyleGuide-DSpaceJavaStyleGuide(Adopted2018,forDSpace7.xandabove) may be useful.

I don't think there is a guideline for this, but personally I tend to add @author only when I was involved in creating the file -- probably because my IDE does it automagically and I have no reason to take it out.  I'm too lazy to add one manually to existing code that I'm updating, unless I've been asked to.

``git`` keeps track of our contributions whether we tag the code or not.

Reply:
Thanks @mwoodiupui that's helpful to know.  I think the path of least resistance makes sense here!